pgcontents/tests/test_pgmanager.py

```diff
@@ -185,6 +185,50 @@ def test_get_file_id(self):
         cm.rename(path, updated_path)
         self.assertEqual(id_, cm.get_file_id(updated_path))
 
+    def test_rename_file(self):
+        cm = self.contents_manager
+        nb, nb_name, nb_path = self.new_notebook()
+        nb_model = cm.get(nb_path)
+        assert nb_model['name'] == 'Untitled.ipynb'
+
+        # A simple rename of the file within the same directory.
+        file_id = cm.get_file_id('Untitled.ipynb')
+        renamed = cm.rename_file(nb_path, 'new_name.ipynb')
+        assert renamed == 1
+        assert cm.get('new_name.ipynb')['path'] == 'new_name.ipynb'
+        assert cm.get_file_id('new_name.ipynb') == file_id
+
+        # The old file name should no longer be found.
+        with assertRaisesHTTPError(self, 404):
+            cm.get('Untitled.ipynb')
+
+        # Test that renaming outside of the root fails.
+        with assertRaisesHTTPError(self, 404):
+            cm.rename_file('../foo', '../bar')
+
+    def test_move_file(self):
```

Comment:
They're different functionalities (enough so that I figured moving deserved its own test) but they happen to use the same function. Do you think I should include this all under `test_rename_file`?